exporter/syslogexporter/sender.go

```diff
@@ -78,8 +78,8 @@ func (s *sender) dial() error {
 		s.conn = nil
 	}
 	var err error
-	if s.tlsConfig != nil {
-		s.conn, err = tls.Dial("tcp", s.addr, s.tlsConfig)
+	if s.tlsConfig != nil && s.network == "tcp" {
```

Comment:
nit: maybe a constant for "tcp" would be ideal

Reply:
I'll open a new pull request to change tcp/udp strings into constants, we use these strings in several places.